client/src/components/AutocompleteText.js

```diff
@@ -0,0 +1,79 @@
+import React, {Component} from 'react';
+import '../css/AutocompleteText.css';
+import escapeStringRegexp from 'escape-string-regexp';
+import  getitems   from './suggestions';
+export default class AutocompleteText extends Component {
+    constructor(props){
+        super(props)
+        // eslint-disable-next-line immutable/no-mutation
+        this.state = {
+            suggestions : [],
+            text : '',
+            filter : '',
```

Comment:
The state `filter` is set in line 18 but besides that, it is not used anywhere in the component. 

Remove the unused state.